aten/src/ATen/native/quantized/QTensor.cpp

```diff
@@ -115,5 +115,19 @@ Tensor& set_quantizer_(Tensor& self, ConstQuantizerPtr quantizer) {
   return self;
 }
 
+Tensor quantized_clone(const Tensor& self) {
+
+  Tensor dst = at::_empty_affine_quantized(
+      self.sizes(),
+      self.options().dtype(self.scalar_type()),
```

Comment:
actually, you don't need to change dtype, just `self.options()` is enough since you are copying